src/platform/user/authorization/containers/MHVApp.jsx

```diff
@@ -52,52 +49,71 @@ export class MHVApp extends React.Component {
     if (this.isIneligible()) { return; }
 
     if (this.needsTermsAcceptance()) {
-      this.props.fetchLatestTerms(TERMS_NAME);
+      const redirectQuery = { tc_redirect: window.location.pathname }; // eslint-disable-line camelcase
+      const termsConditionsUrl = appendQuery('/health-care/medical-information-terms-conditions', redirectQuery);
+      window.location.replace(termsConditionsUrl);
```

Comment:
This goes to the T&C page with the current health tool's URL as a query param. That allows acceptance to redirect back here and use another query param as a flag to trigger a success message.